formula_renames.json

```diff
@@ -6,6 +6,7 @@
   "app-engine-java-sdk": "app-engine-java",
   "ark": "velero",
   "arm": "nyx",
+  "azion": "azion-cli",
```

Comment:
```suggestion
  "azion-cli": "azion",
```

Reply:
I've looked at other examples in this file and it seemed like the name in quotes was the same as the one found in the Formula folder, that's why I've written it like this.

Reply:
The old name comes first. Here's an example running `brew info` on the formula above yours:
```
brew info arm
Warning: Formula arm was renamed to nyx.
```

Reply:
Ok, so I need to rename my formula from azion-cli.rb to azion and then add the information to this file. 
I'll carry out all the changes. Thanks